src/source/Ice/Network.c

```diff
@@ -182,6 +175,44 @@ STATUS createSocket(PKvsIpAddress pHostIpAddress, PKvsIpAddress pPeerAddress, KV
         CHK(FALSE, STATUS_SOCKET_SET_SEND_BUFFER_SIZE_FAILED);
     }
 
+    *pOutSockFd = (INT32) sockfd;
+
+#ifdef _WIN32
+    UINT32 nonblock = 1;
```

Comment:
This might cause a compiler warning/error. Perhaps, move this in the beginning of the function. If non-Windows compilers complain that it's not used you could simply add UNUSED_PARAM(nonblock)

Reply:
this is not causing any warnings in non-windows compiler.